clients/application-kafka/src/main/java/org/eclipse/hono/application/client/kafka/impl/KafkaApplicationClientImpl.java

```diff
@@ -102,6 +103,11 @@ public KafkaApplicationClientImpl(
         this.consumerConfig = consumerConfig;
     }
 
+    @Override
+    public void addOnClientReadyHandler(final Handler<AsyncResult<Void>> handler) {
```

Comment:
is this method being used anywhere?

Reply:
Not in Hono itself. But it is useful for applications using the Hono (Kafka)ApplicationClient.